johnsonandjohnson/Bodiless-JS
# feat(example/test-site): Updated the test site to have the correct pa…

…dding in the flexbox example





## Changes


## Test Instructions


Add items to any flexbox on the test site
observe that 
1. there is padding between each item
2. there is no padding around the flexbox

## Related Issues

Fixes #235
